va/va.c

```diff
@@ -741,6 +743,72 @@ static VAStatus va_legacy_opendriver(VADisplay dpy)
     return vaStatus;
 }
 
+static VAStatus va_new_opendriver(VADisplay dpy)
+{
+    VADisplayContextP pDisplayContext = (VADisplayContextP)dpy;
+    /* In the extreme case we can get up-to 5ish names. Pad that out to be on
+     * the safe side. In the worst case, the DRIVER BUG below will print and
+     * we'll be capped to the current selection.
+     */
+    char *drivers[20] = { 0, };
```

Comment:
20 should be enough now, but does we need a check?

Reply:
What check are you looking for - can you provide a concrete example?

Reply:
no need check now, it is impossible that {i915, {iHD, i965, i966 .... iHD1, iHD2...} exceed 20 backend UMD drivers for one KMD.